File: sampleapp/app/src/main/java/com/mwping/android/plugin/sampleapp/LogUtils.java
package com.mwping.android.plugin.sampleapp;

/**
 * weiping@atlasv.com
 * 2022/3/21
 */
class LogUtils {
    public static LogTrace start(String className, String methodName) {
        return new LogTrace(className, methodName).start();
    }
}

File: sampleapp/app/src/main/java/com/mwping/android/plugin/sampleapp/LogUtils.kt
package com.mwping.android.plugin.sampleapp

/**
 * weiping@atlasv.com
 * 2022/3/21
 */
object LogUtils {
    @JvmStatic
    fun start(className: String, methodName: String): LogTrace {
        return LogTrace(className, methodName).start()
    }
}